src/OpenTelemetry/BatchExportProcessorOptions.cs

```diff
@@ -38,5 +84,24 @@ public class BatchExportProcessorOptions<T>
         /// Gets or sets the maximum batch size of every export. It must be smaller or equal to MaxQueueLength. The default value is 512.
         /// </summary>
         public int MaxExportBatchSize { get; set; } = BatchExportProcessor<T>.DefaultMaxExportBatchSize;
+
+        private static bool TryLoadEnvVarInt(string envVarKey, out int result)
```

Comment:
It seems that the logic will be used in many other places. Consider extracting the common logic (e.g. BatchExportProcessor doesn't understand environment variable, it simply "retrieves" key-value pair configuration from an abstract configuration provider, and the default provider is environment variable).

Reply:
Feel a little YAGNI. I will try making a dedicated PR even though I think it will be not worth adding an additional layer of interaction.

Still, I think it may be worth to rename `TryLoadEnvVarInt` and put it into some helper class to reuse it in other places.

Reply:
Would it be acceptable to use `OpenTelemetrySdkEventSource` in `OtlpExporterOptions` and `JaegerExporterOptions` instead of the currently used `OpenTelemetryProtocolExporterEventSource` and `JaegerExporterEventSource` to indicate problems when reading and parsing the env vars?